src/removed_functions.rs

```diff
@@ -0,0 +1,21 @@
+use pgrx::prelude::*;
+
+// We need to keep exposing linkable wrappers for old, removed functions. This
+// is required so that transitory version upgrades work as expected.
+//
+// For example, assume the following update chain:
+// 0.22 -> 0.23 -> 0.24 -> 0.25 -> 0.26
+// In the version 0.23, `pgmq_archive` was added. But if the user has the dynamic
+// library from version 0.26, the transitory update from 0.22 to 0.23 won't work
+// as postgres can see that `pgmq_archive` isn't available in the dynamic library.
+// For this reason, we need to keep `pgmq_archive` in the dynamic library forever
+// to keep update compatibility.
+//
+#[pg_extern(sql = "")]
+pub fn pgmq_read() {}
+#[pg_extern(sql = "")]
+pub fn pgmq_read_with_poll() {}
+#[pg_extern(sql = "")]
+pub fn pgmq_archive() {}
+#[pg_extern(sql = "")]
+pub fn pgmq_archive_batch() {}
```

Comment:
i wonder if there is a way for us to drop these with the 1.0 release - I think some manual steps to upgrade from <1.0 -> 1.0 would be ok. I am not sure how we can get around this though.

Reply:
We can ask to update to 0.x (the latest 0 dot something version) and then to 1.0, and you then wouldn't need those